saleor/graphql/page/types.py

```diff
@@ -0,0 +1,18 @@
+from graphene import relay
+
+from ...page import models
+from ..core.types import CountableDjangoObjectType
+
+
+class Page(CountableDjangoObjectType):
+    class Meta:
+        model = models.Page
+        interfaces = [relay.Node]
+
+
+def resolve_pages(info):
+    return models.Page.objects.public().distinct()
+
+
+def resolve_all_pages(info):
```

Comment:
Maybe this would fit better in `/dashboard/graphql/`, since this is regular resolver but for the dashboard API?